inf-clojure.el

```diff
@@ -559,6 +575,26 @@ The value is nil if it can't find one."
   "Return the name of the symbol at point, otherwise nil."
   (or (thing-at-point 'symbol) ""))
 
+(defun inf-clojure--empty-string-p (string)
+  "Return true if the string is empty or nil.  Expects STRING."
+  (or (null string)
+      (zerop (length string))))
+
+(defun inf-clojure--read-classpath (classpath-option)
+  "Read the classpath from the input CLASSPATH-OPTION."
+  (let ((classpath-file-path (concat (inf-clojure-project-root) classpath-option)))
+    (cond
+     ((and (file-exists-p classpath-file-path) (file-readable-p classpath-file-path))
+      (f-read classpath-file-path))
+     ;; TODO launch a command that returns the classpath string?
+     ((message (concat "Option \"" classpath-option "\" was not a (readable) file, the classpath will be empty."))
+      ""))))
+
+(defun inf-clojure--maybe-set-program (program)
```

Comment:
I added it because I though you'd want to modify the `inf-clojure-program` only if it is set as a command , not if it is a cons cell, which indicates you don't want to launch anything...open to other ideas..